community/dicordeiro.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
# Olá, sou o Diego Cordeiro. 
 <div id="badges">
  <a href = "https://www.linkedin.com/in/diego-cordeiro-4628a187">
    <img src="https://img.shields.io/badge/LinkedIn-blue?style=for-the-badge&logo=linkedin&logoColor=white" alt="LinkedIn Badge"/>
  </a>
</div>

Sou Analista de Sistemas Sr. e integro um grupo de Analistas Funcionais e Líderes Técnicos na Telefonica (Vivo).

Tenho experiência na implementação de Projetos de Alta Complexidade, atuando com várias ferramentas como a ServiceNow (ITBM / ITSM / SOM / Performance Analytics) , SalesForce, SAP Ecommerce (HYBRIS) , AEM.


[ GitHub](https://github.com/dicordeiro)

## 💡 Habilidades

![HTML5](https://img.shields.io/badge/HTML5-000?style=for-the-badge&logo=html5)
![CSS3](https://img.shields.io/badge/CSS3-000?style=for-the-badge&logo=css3&logoColor=264CE4)

## Aprendendo
 <img align="center" alt="Rafa-PHP" height="30" width="40" src="https://raw.githubusercontent.com/devicons/devicon/master/icons/python/python-original.svg"> <img align="center" alt="Gu-MySQL"  height="30" width="40" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/mysql/mysql-original-wordmark.svg">

## GitHub Stats

![GitHub Stats](https://github-readme-stats.vercel.app/api?username=dicordeiro&theme=transparent&bg_color=000&border_color=30A3DC&show_icons=true&icon_color=30A3DC&title_color=E94D5F&text_color=FFF&hide=stars)